### ISSUE TITLE
Casting Basin and Steam hammer are asking steel block in pack without steel

### ISSUE THREAD

**Issue by Kriiks** [TeamOpenIndustry/ImmersiveRailroading#774]
IR asks steel block in modpack without steel as a material in pack.
MC 1.12.2
IR 1.7.3

We played 3 packs in row with heavy IE usage we decided to make a pack without it and were happy to see that Railroading offers alternative recipes without IE. All was good till we started to build Casting Basin. All other IE specific materials are swapped, but there is still one steel block requirement. We litarally don't have steel in pack at all, so we can't even cheat it in. So now we are completely stuck.
Checked now all multiblock machines recipes.. steam hammer aslo asks for a block of steel.

![javaw_22bWLaMxHD](https://user-images.githubusercontent.com/16120946/79070869-acacb000-7cd8-11ea-9f0a-0c0f77df5bc8.png)

**Comment by cam72cam** [comment]
It will only ask for steel if it exists in the pack in some form.  Then name "Block of Steel" comes from some other mod IR detected that has a steel block.  What is your mod list?

**Comment by Kriiks** [comment]
Was looking now through everything and comes out that our culprit is Chisel. If I was just searching for steel, it didn't show up, but when I was browsing through Chisel blocks, I found chiseled versions of steel.
Really annoying it still lists it, even if there is no way to actually produce steel. Could it be possible for IR just ignore Chisel?

**Comment by cam72cam** [comment]
Not really, does Chisel have a config setting you can use?

**Comment by Kriiks** [comment]
Had a wonderful day without internet, but yes: Chisel has a configuration option to turn blocks on and off indeed. After I found out what mod is causing our weird issue and having several hours of brain lag, I went to check Chisel config. 
So even if I find it really odd that mod what does not let you to produce steel, is making IR to ask for steel blocks, I can't really call it technically a bug because steel block (in a hidden way) was still listed.
Apologies for listing it as a bug, but it was quite unexpected :)
